Framework/Backend/services/kafka.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+const { Kafka, logLevel } = require('kafkajs')
+const WebSocketMessage = require('../websocket/message.js');
+const log = new (require('./../log/Log.js'))(`${process.env.npm_config_log_label ?? 'framework'}/kafka`);
+const url = require('url')
+
+/**
+ * Gateway for all Kafka notification service
+ */
+class KafkaConnector {
+  /**
+   * Prepares connector
+   * @param {object} config Config with list of Kafka brokers
+   */
+  constructor(config) {
+    if (!config) {
+      log.warn('Missing configuration');
```

Comment:
Should we throw an error here as well to be in line with line 33 where if there are missing brokers we throw error?

Reply:
In this case we might not want to throw as it represents case when configuration is missing, therefore Kafka should be disabled.